compiler/crates/relay-transforms/src/validations/validate_module_names/extract_module_name.rs

```diff
@@ -161,6 +165,10 @@ mod tests {
             extract_module_name("/path/non-numeric-end-.js"),
             Some("nonNumericEnd".to_string())
         );
+        assert_eq!(
+            extract_module_name("/path/__tests__/index.test.js"),
+            Some("Tests".to_string())
+        );
```

Comment:
can you add a test case for a path like `/path/button/indexButton.js`?

Reply:
done